.github/workflows/rpm.yml

```diff
@@ -0,0 +1,37 @@
+name: Build_rpm
+on:
+  push: 
+    tags:
+      - 'v*'
```

Comment:
I don't believe this will pick up releases following the current tag format: `ec2rl-v#.#.#`

Reply:
You're correct. Will change to accept that format.